### ISSUE TITLE
Get the results of other layers into loss function

### ISSUE THREAD

**Issue by nicolalandro** [keras-team/keras#13371]
It is possible to obtain the values of another layer into lossfunction? For example for some classification problem if you want to use center loss, and crossentropy you minimize the crossentropy on the last layer but the centerloss work with the feature, so it need the penultimate layer.
Up to now I use Tensorflow or Pythorch because can't do this with keras. It is not possible realy?
